### DISCUSSION TITLE
vue2.0 use atropos

### DISCUSSION THREAD

**Discussion by ghost** [nolimits4web/atropos#22] [Q&A]
atropos/vue in ./node_modules/cache-loader/dist/cjs.js??ref--12-0!./node_modules/babel-loader/lib!./node_modules/cache-loader/dist/cjs.js??ref--0-0!./node_modules/vue-loader/lib??vue-loader-options!./src/layouts/basic.vue?vue&type=script&lang=js&

**Comment by wuhou123** [comment]
me too,so hard to use,i am so sad

**Comment by RenHans** [comment]
me three,vue2 can use it? someBody success?
